articles/introduction-to-sequalize-orm-for-nodejs/index.md

```diff
@@ -0,0 +1,215 @@
+
+SQL enables us to perform operations abbreviated as `CRUD`; Create Read Update and Delete. To illustrate this we have two tables in our database `Employees`:
+
+Table `lego_people`
+
+|name   |age    |
+|-----  |----   |
+|james  |34     |
+|craig  |15     |
+|paul   |67     |
+|cynthia|55     |
+
+
+Table `lego_height`
+
+|name   |cm     |
+|-----  |----   |
+|james  | 4     |
+|craig  | 5     |
+|paul   | 7     |
+|cynthia| 5     |
+
+So if we want to `create` data in the `lego_people` table we:
+
+```sql
+INSERT INTO lego_people(name, age)
+VALUES ('Joe' ,12);
+```
+The `INSERT INTO` statement is used to create data in a database table. We pass in `name` and `age` as arguments to specify the properties we want to add.
+To `read` value from the `lego_people` table:
+
+```sql
+SELECT name FROM lego_people;
+```
+The `name` represents the column name in the` lego_people` table. The `SELECT` statement is used to read data from the database.
+
+To `update` a record in the `lego_height` table by making Paul 8cm tall:
+
+```sql
+UPDATE lego_height
+SET name = 'paul', height = 8;
+```
+
+We can also perform a `delete` operation from the database. For example, let's delete `paul` altogether using the following command:
+
+```sql
+DELETE FROM lego_height
+WHERE name = 'paul';
+```
+
+The `WHERE` clause specifies the data to be deleted.
+
+If for instance, we wanted to have the records of both age and height in one table we would use a `JOIN` statement to combine both tables:
+
+```sql
+SELECT name, age FROM lego_people
+LEFT JOIN lego_height USING (name);
+```
+
+This would result to this:
+
+|name   |age   |cm    |
+|-----  |----- |----- | 
+|james  |34    | 4    |    
+|craig  |15    | 5    |
+|paul   |67    | 7    |    
+|cynthia|55    | 5    |
+
+### Object Relational Mappers(ORM) Node.js Approaches
+Before we get started with Sequelize, let's get to understand what an ORM is and some of the supported ORMs in Node.js. Object Relational Mapping is a simplified way of converting data between relational databases and objects (Primrose katena, 2020).
+
+Most times we encounter scenarios where we have to write complex SQL query statements to perform CRUD operations from a database. Worse enough, we still need to convert data from the database into an object that is compatible with our language of choice as we will see in the next chapter. It additionally simplifies the manipulation of database engines whenever the need arises. 
+
+So how does ORMs work, well first, choosing the right database you want to use. Then decide on the Object Relational Mapper that would work efficiently with the database and install it. For our case since we're using Node.js, we'll consider Sequelize.
+Second, we create a database model. This represents the structure of our database. Database models provide the advantage of scalability and efficiency. The structure defined in the model includes tables, database collection, and columns. We then connect our database.
```

Comment:
```suggestion
So how does ORMs work, well first, choosing the right database you want to use. Then decide on the Object Relational Mapper that would work efficiently with the database and install it. For our case since we're using Node.js, we'll consider Sequelize.

Second, we create a database model. This represents the structure of our database. Database models provide the advantage of scalability and efficiency. The structure defined in the model includes tables, database collection, and columns. We then connect our database.
```